promscale/installation/source.md

```diff
@@ -0,0 +1,80 @@
+# Promscale from source
```

Comment:
This page doesn't mention `promscale_extension` at all. I am aware that the from-source installation for the extension is a huge PITA, but I also find it surprising that it's not mentioned at all.

Reply:
Overall the instructions on this page are confusing. The two sections have duplicate content. I think that we don't need steps 1 and 2 in `Installing the remote storage connector` (technically, those steps are a duplicate of steps 1 and 2 in `Installing Promscale from source`, and are the _actual_ 'from source' installation instructions, to make things even more confusing 😂 ).

Reply:
Good catch! Changing the complete content of `source.md`